ra-optimisation/rao-api/src/main/java/com/powsybl/openrao/raoapi/parameters/LoadFlowAndSensitivityParameters.java

```diff
@@ -58,7 +59,10 @@ public double getSensitivityFailureOvercost() {
     }
 
     public void setSensitivityFailureOvercost(double sensitivityFailureOvercost) {
-        this.sensitivityFailureOvercost = sensitivityFailureOvercost;
+        if (sensitivityFailureOvercost < 0) {
+            BUSINESS_WARNS.warn("The value {} for `sensitivity-failure-overcost` is smaller than 0. This would encourage the optimizer to make the loadflow diverge. Thus, it will be set to + {}", sensitivityFailureOvercost, -sensitivityFailureOvercost);
```

Comment:
throw exception instead (we do it elsewhere when the parameter is wrong, for example in setPstSensitivityThreshold)

Reply:
I talked about it with Philippe and we copied the logic of the method `setCurativeMinObjImprovement` in class `ObjectiveFunctionParameters`. I can modify the code but I prefer that you discuss it with him before.

Reply:
ok then, for now. but i think we should harmonize the parameters more, and see what is done in other powsybl projects